src/nft/components/card/containers.tsx

```diff
@@ -30,34 +31,41 @@ const StyledDetailsContainer = styled(Column)`
   }
 `
 
+const StyledActionContainer = styled.div`
+  position: absolute;
+  bottom: -32px;
+  left: 8px;
+  right: 8px;
+  transition: ${({ theme }) =>
+    `${theme.transition.duration.medium} ${theme.transition.timing.ease} bottom, ${theme.transition.duration.medium} ${theme.transition.timing.ease} visibility`};
+  will-change: transform;
+  visibility: hidden;
+
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
+    visibility: visible;
+    bottom: 8px;
+  }
+`
+
+const ActionContainer = ({ children }: { children: ReactNode }) => (
```

Comment:
confused what this is doing - isn't this the same thing as just `StyledActionContainer` by itself?